lang/fr.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+$lang['title'] = 'TecArt Ticket System';
+
+$lang['foot_note'] = 'Ticket-Web Interface &copy; by TecArt GmbH';
+
+$lang['custom_number']  = 'N ° de client';
```

Comment:
This looks a little odd with the space and the degree(?) symbol. I'm not as good with french i10n but wouldn't it be better to just write 'numéro' as full word?

Reply:
It's common french, but should be "N° de client".